fuse/file.go

```diff
@@ -36,7 +37,18 @@ func (fi *File) Attr(ctx context.Context, attr *fuse.Attr) error {
 	}
 	debugLog("exec file attr: %v", fi.path)
 
-	attr.Mode = 0755
+	var filePerm os.FileMode = 0640
+	attr.Mode = filePerm 
+	if fi.m.options.Offline {
+		fmt.Printf("info %v\n", info)
+		isCached, err := fi.m.fs.IsCached(fi.path)
+		if err != nil || !isCached {
```

Comment:
This will also show files as symlinks where ``IsCached`` errored for some reason. We should at least log the error.

Reply:
Done with 3a5c46b

I have a question. The fuse files avoid using log facilities, i.e. `log.Debug()` and sparingly use `debugLog()`. Is there a deep reason for it? Is it used to enable/disable logging only for fuse module with `enableDebugLogs = false/true`  https://github.com/sahib/brig/blob/6140e52ffe5713c50f2ea01bb0cf9a0f0e69e799/fuse/fs.go#L15

Reply:
No, I don't think there was a deeper reasons to it. FUSE debugging can get very spammy, so I wanted an easy flag I can toggle. Some day I will implement proper subsystem logging.